qualtran/bloqs/factoring/mod_mul.py

```diff
@@ -215,13 +218,13 @@ def _generalize_k(b: Bloq) -> Optional[Bloq]:
     return b
 
 
-@bloq_example(generalizer=(ignore_split_join, ignore_alloc_free, _generalize_k))
+@bloq_example(None, generalizer=(ignore_split_join, ignore_alloc_free, _generalize_k))
```

Comment:
is this required? Kindof ruins the `~*aesthetic*~`

Reply:
No, this was my first attempt, but the root problem was that the override in bloq_example was missing a default.  I missed reverting these after I discovered that.

Reply:
cool, that's what I was hoping you'd say :)